src/MOI_wrapper.jl

```diff
@@ -196,25 +196,58 @@ const _SUPPORTED_MODEL_ATTRIBUTES = Union{
     # MOI.RawStatusString,  # TODO
     MOI.ResultCount,
     # MOI.Silent, # TODO
-    # MOI.TerminationStatus,
+    MOI.TerminationStatus,
     # MOI.PrimalStatus,
     # MOI.DualStatus
 }
-
+@enum HiGHS_ModelStatus begin
+    NOTSET 
```

Comment:
Maybe for clarity, have `NOTSET = 0` to indicate the enum starts at 0 (even though this is the default).